.gitignore

```diff
@@ -1,4 +1,7 @@
 Rust/*/target/
+Nim/*/*
+!Nim/*/*.*
+!Nim/*/*/
```

Comment:
Isn't there a way to simplify this?
What paths are we trying to ignore here?

Reply:
I think I can put a local config in the nim dir to override this. I will have to investigate a bit haven't used nim in forever.

Reply:
Default behavior will make a Nim/fibonacci/fibonacci bin or something else if you compile to js etc